service/awsconfig/v8/resource/cloudformation/update.go

```diff
@@ -20,16 +20,16 @@ func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateChange inte
 	}
 
 	if updateStackInput.StackName != nil && *updateStackInput.StackName != "" {
-		r.logger.LogCtx(ctx, "debug", "updating main stack")
+		r.logger.LogCtx(ctx, "level", "debug", "message", "updating the guest cluster cloud formation stack")
```

Comment:
Do we need cloud formation in all the messages? Is the resource name included in the logging?

Some of these are quite long so I just want to be sure they are readable in logs output.

Reply:
How should the wording be? Should it just be `main stack` instead of `guest cluster cloud formation`? Or should it be `guest cluster main stack`? The latter is not really shorter though. I personally do not worry about the length as long as it is one informative message/sentence and not some kind of novel. When following logs the log lines are anyway too long to have one log per line and this is also where EFK kicks in for us.

Reply:
`guest cluster main stack` is shorter and makes sense to me. My issue is not so much length as "cloud formation" is redundant because it's the cloudformation resource.

Reply:
Ok cool. Going with your suggestion. Done.